specification/loadtestservice/data-plane/readme.md

```diff
@@ -26,6 +26,45 @@ openapi-type: data-plane
 tag: package-2022-11-01
 title: LoadTestingClient
 ```
+### Tag: package-2025-07-01-preview
```

Comment:
I think this file should be removed if you have the readme.md files in each of subfolder.